unix/vncpasswd/vncpasswd.cxx

```diff
@@ -109,10 +151,22 @@ static std::vector<uint8_t> readpassword() {
     std::string first = passwd;
     if (first.size() < 6) {
       if (first.empty()) {
-        fprintf(stderr,"Password not changed\n");
+        fprintf(stderr,_("Password not changed\n"));
         exit(1);
       }
-      fprintf(stderr,"Password must be at least 6 characters - try again\n");
+      fprintf(stderr,_("Password must be at least 6 characters - try again\n"));
+      continue;
+    }
+
+    if (first.size() > 8) {
+      fprintf(stderr,_("Password should not be greater than 8 characters\nBecause only 8 valid characters are used - try again\n"));
```

Comment:
It's best to avoid formatting in the strings as it makes it difficult to translate.